website_sale_category_breadcrumb/views/templates.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template id="breadcrumb" inherit_id="website_sale.product" priority="999">
+        <xpath
+            expr="//ol[hasclass('breadcrumb')]/li[@t-if='category']"
+            position="replace"
+        >
+            <t
+                t-if="category"
+                t-call="website_sale_category_breadcrumb.breadcrumb_recursive"
+            />
+        </xpath>
+    </template>
+    <template id="breadcrumb_recursive">
+        <t t-if="category.parent_id">
+            <t t-call="website_sale_category_breadcrumb.breadcrumb_recursive">
+                <t t-set="category" t-value="category.parent_id" />
+            </t>
+        </t>
+        <li class="breadcrumb-item">
+            <a
+                t-att-href="keep('/shop/category/%s' % slug(category), category=0)"
+                t-field="category.name"
+            />
+        </li>
+    </template>
+    <template
+        id="additional_breadcrumbs"
+        inherit_id="website_sale.product"
+        name="Show breadcrumbs for additional categories"
+        customize_show="True"
+        active="True"
+    >
+        <xpath
+            expr="//section[@id='product_detail']/div[hasclass('row')]"
+            position="inside"
+        >
+            <div class="col-md-12">
+                <t
+                    t-if="product.public_categ_ids and ((not category) or (not category in product.public_categ_ids) or (category in product.public_categ_ids and len(product.public_categ_ids) &gt; 1))"
+                >
+                    <p
+                        class="info"
```

Comment:
Just a small suggestion. We could improve the design a little bit so that it doesn't look so close to the search bar.
![image](https://github.com/user-attachments/assets/17972f64-64f4-48df-b086-80b5fba2c9fc)

```suggestion
                        class="info mt-3 mb-0"
```

![image](https://github.com/user-attachments/assets/da125997-cf2c-470f-836f-cf1ca3eb0e43)

Reply:
done